plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeMetadata.java

```diff
@@ -3113,6 +3113,10 @@ private Optional<SystemTable> getRawSystemTable(ConnectorSession session, Schema
                     systemTableName,
                     getCommitInfoEntries(tableLocation, session),
                     typeManager));
+            case PROPERTIES -> Optional.of(new DeltaLakePropertiesTable(
+                    systemTableName,
+                    transactionLogAccess.getMetadataEntry(tableSnapshot, session),
```

Comment:
There's another `getMetadataEntry` call in this method. Let's define a variable and use it here.